src/govuk/components/input/_index.scss

```diff
@@ -91,4 +91,88 @@
     max-width: 5.4ex;
   }
 
+  .govuk-input__wrapper {
+    display: flex;
+
+    .govuk-input {
+      flex: 0 1 auto;
+    }
+
+    .govuk-input:focus {
+      // Hack to stop focus style being overlapped by the suffix
+      z-index: 1;
```

Comment:
I'm not keen on this fix but I also can't find another way of stopping the suffix overlapping the input focus border.